code/lang/TTTLanguage.cs

```diff
@@ -1,70 +1,66 @@
-using System;
 using System.IO;
 using System.Collections.Generic;
 
 using Sandbox;
 
-namespace TTTReborn.Language
+namespace TTTReborn.Globalization
 {
     public static class TTTLanguage
     {
-        // TODO: On cvar change callback
-        [ClientVar("ttt_language", Help = "Your current language", Saved = true)]
-        public static string ActiveLanguage { get; set; }
+        public const string FALLBACK_LANGUAGE = "en-US";
 
-        [ServerVar("ttt_server_language", Help = "Server's current language", Saved = true)]
-        public static string ServerLanguage { get; set; }
-        public const string FALLBACK_LANGUAGE = "English";
-
-        public static Dictionary<string, TLanguage> Languages = new();
-
-        private static void FallbackLanguage()
+        public static Language ActiveLanguage
         {
-            if (Host.IsClient)
+            get => _activeLanguage;
+            set
             {
-                ActiveLanguage = FALLBACK_LANGUAGE;
+                _activeLanguage = value ?? GetLanguageByName(FALLBACK_LANGUAGE);
             }
+        }
+        private static Language _activeLanguage;
 
-            if (Host.IsServer)
+        public static Language ServerLanguage
+        {
+            get => _serverLanguage;
+            set
             {
-                ConsoleSystem.SetValue("ttt_server_language", FALLBACK_LANGUAGE);
+                _serverLanguage = value ?? GetLanguageByName(FALLBACK_LANGUAGE);
             }
         }
+        private static Language _serverLanguage;
+
+        public static readonly Dictionary<string, Language> Languages = new();
 
         public static void LoadLanguages()
-        {            
-            foreach ( string file in FileSystem.Mounted.FindFile("/lang/packs/", "*.json", false) )
+        {
+            foreach (string file in FileSystem.Mounted.FindFile("/lang/packs/", "*.json", false))
```

Comment:
I forgot to point out that this doesn't support hot loading. We need to figure out how to do this, that'll be really great.

Reply:
Afaik, hotloading won't trigger for .json files, that's why I don't think we need hotloading.